content/blog/summary-from-the-9th-annual-chapel-implementers-and-users-workshop-chiuw-2022.md

```diff
@@ -0,0 +1,73 @@
+
+In this post, we summarize highlights from the recent [9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)](https://chapel-lang.org/CHIUW2022.html).  We are the general chair of the workshop, Michelle Strout, and the program committee chair, Engin Kayraklioglu, and members of the Chapel development team at HPE.  Read on to hear about some exciting applications that are using Chapel productively, the coding day that happened the day before the workshop, updates on the project, and feedback the team received during and after the workshop.
+
+**Applications Written in Chapel**
+
+Programmers generally enjoy trying out new programming languages, but before they lean on a programming language it is always better to see example use cases.  The Chapel programming language is being used productively in a range of application domains: data science, aeronautical simulations, cosmology simulations, and quantum diagonalization to name a few. CHIUW featured the following talks on different Chapel applications:
+
+* Large-Scale and User-Friendly Exact Diagonalization in Chapel (<https://youtu.be/vBxPTzIRRr0>)
+* Recent Developments in the CHApel Multi-Physics Simulation Software (<https://youtu.be/uTE_RZkODOk>)
+* UltraLight Dark Matter in Simulations: A Chapel-Powered Eigenstate Perspective (<https://youtu.be/YrXYpgnt4rQ>)
+* Implementing and Optimizing Parquet I/O in Chapel (<https://youtu.be/pstRsgMhCDA>)
+* Truss Analytics Algorithms and Integration in Arkouda (<https://youtu.be/xI9EByv7A5M>)
+* From C and Python to Chapel as My Main Programming Language (<https://youtu.be/400jmMzdzHQ>)
+
+**Coding Day**
+
+For coding day, anyone interested in working one-on-one or in small groups with developers from the Chapel team at HPE could indicate their interest in an online form.  We had 7 different sessions that occurred.  Programmers interested in Chapel were able to ask questions specific to their Chapel code, interactively make changes to the code, and work through issues with Chapel developers around for immediate assistance. In these sessions, Chapel users and developers worked on
+
+* Adding diagnostic support for Chapel runtime’s remote data cache
+* Writing a cell-list module/library in Chapel
+* Investigating porting a Dask application to Arkouda
+* Implementing a Lisp interpreter in Chapel
+* Optimizing distributed memory performance of a very large-scale matrix-vector multiplication
+* Discussing Chapel’s nascent GPU support and going over the internals of the current implementation
+* Learning Chapel in a peer-programming setting
+
+Next year, we plan to keep coding day virtual and grow it to include more small groups at non-intersecting times and a publicized schedule ahead of time.  Email Engin at [engin@hpe.com](mailto:engin@hpe.com) if you have any thoughts about what you would like to work on or see during next year’s Chapel coding day.
```

Comment:
Recommend using initial caps on "coding day" so it's "Coding Day" in both instances. 

"...a publicized schedule ahead of time" requires some grammatical fixes - I'd recommend rephrasing it a bit so the whole sentence reads:

"Next year, we plan to keep Coding Day virtual and grow it to include more small groups at non-intersecting times, and publish a schedule ahead of time."